interfaces/builtin/pulseaudio_test.go

```diff
@@ -39,27 +39,42 @@ var _ = Suite(&PulseAudioInterfaceSuite{
 	iface: &builtin.PulseAudioInterface{},
 })
 
+const paMockPlugSnapInfoYaml = `name: other
+version: 1.0
+plugs:
+ pulseaudio:
+  interface: pulseaudio
+apps:
+ app2:
+  command: foo
+  plugs:
+   - pulseaudio
+`
+
+const paMockSlotSnapInfoYaml = `name: pulseaudio
+version: 1.0
+slots:
+ pulseaudio:
+  interface: pulseaudio
+apps:
+ app1:
+  command: foo
+  slots:
+   - pulseaudio
+`
+
+const paMockSlotOSSnapInfoYaml = `name: pulseaudio
```

Comment:
Hm, the name is slightly confusing `paMockSlot*OS*SnapInfoYaml` but `name: pulseaudio`